### PR TITLE
update scheme format

### PR DESCRIPTION
**Pull request by SelimGribaa**
Asana task : https://app.asana.com/0/1141396442496489/1181397055064779

fix Redirection scheme Android SDK and update regex of WebViewClient

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -33,9 +33,10 @@
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
                 <data
-                        android:scheme="@string/reachfive_scheme"
-                        android:path="callback"
-                        tools:ignore="AppLinkUrlError" />
+                    android:scheme="@string/reachfive_scheme"
+                    android:host="callback"
+                    android:pathPrefix="/LSCdhBOtrPeh8RFLz6bb"
```
Comment (roxanemace) [inline app/src/main/AndroidManifest.xml:38]: Can we create a resource variable for the client ID in the `settings.xml` file?

**Comment by roxanemace** [comment]
The `README.md` must be updated :)

File: README.md [hunk 2]
```diff
@@ -29,13 +29,15 @@ CLIENT_ID=my-reachfive-client-id
...
+The URL scheme must follow this pattern: `reachfive://callback/${clientId}`.
 
 To login with a WebView, the scheme's path must also be set in a resource file as below:
 
 ```xml
 <resources>
-    <string name="reachfive_scheme">reachfive-${clientId}</string>
+    <string name="reachfive_scheme">reachfive</string>
+    <string name="reachfive_client_id">/${clientId}</string>
+
```
Comment (roxanemace) [inline README.md:40]: Saut à la ligne inutile

**Review by roxanemace — APPROVED** [review]

File: app/src/main/res/values/settings.xml [hunk 3]
```diff
@@ -2,5 +2,7 @@
 <resources>
     <string name="facebook_app_id">2326266780745811</string>
     <string name="fb_login_protocol_scheme">fb2326266780745811</string>
-    <string name="reachfive_scheme">reachfive-lscdhbotrpeh8rflz6bb</string>
+    <string name="reachfive_scheme">reachfive</string>
+    <string name="reachfive_client_id">/LSCdhBOtrPeh8RFLz6bb</string>
```
Comment (StanWarz) [inline app/src/main/res/values/settings.xml:6]: This / should not belong here.

File: sdk-webview/src/main/java/com/reach5/identity/sdk/webview/ReachFiveLoginActivity.kt [hunk 4]
```diff
@@ -72,7 +72,8 @@ class ReachFiveLoginActivity : Activity() {
 
     inner class ReachFiveWebViewClient : WebViewClient() {
 
-        private val PATTERN = Pattern.compile("^(reachfive-\\w+:\\/\\/callback)(.*)$")
+        // regex : (reachfive://callback/word character: [a-zA-Z_0-9])(any character zero or more times)
```
Comment (StanWarz) [inline sdk-webview/src/main/java/com/reach5/identity/sdk/webview/ReachFiveLoginActivity.kt:75]: I think the comment `// regex : reachfive://callback/word` is clear enough =)

**Review by StanWarz — CHANGES_REQUESTED** [review]
Request about the scheme pattern.

**Review by StanWarz — APPROVED** [review]

**Pull request merged**